dashboard/config/routes.rb

```diff
@@ -111,7 +111,7 @@
     end
   end
 
-  get '/beta', to: 'home#beta', as: 'beta'
+  get '/beta', to: redirect('/')
```

Comment:
want to write a test? :)


See test/integration/redirects_test.rb, should be pretty easy